src/UnitySampleProject/Assets/Samples/web3.unity SDK/2.5.0/Web3.Unity Samples/Scripts/Scenes/SampleMain/SygmaClient.cs

```diff
@@ -0,0 +1,90 @@
+using System;
+using System.Numerics;
+using System.Threading.Tasks;
+using ChainSafe.Gaming.Evm.Contracts;
+using ChainSafe.Gaming.Evm.Transactions;
+using ChainSafe.Gaming.SygmaClient;
+using ChainSafe.Gaming.SygmaClient.Dto;
+using ChainSafe.Gaming.SygmaClient.Types;
+using ChainSafe.Gaming.UnityPackage;
+using ChainSafe.Gaming.Web3;
+using Nethereum.Hex.HexTypes;
+using Scripts.EVM.Token;
+using UnityEngine;
+using UnityEngine.UI;
+using Environment = ChainSafe.Gaming.SygmaClient.Types.Environment;
+
+[Serializable]
+public class SygmaConfig
```

Comment:
We can move all fields right into the sample behaviour, as this is more like SygmaSampleConfig than the actual config of the SygmaClient.